addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -82,6 +82,9 @@ const PopupWidget = publicWidget.Widget.extend({
      * @private
      */
     _onCloseClick: function () {
+        if (this.$el.find('.media_iframe_video').length) {
+            this.$el.find('iframe')[0].src = this.$el.find('iframe')[0].src;
+        }
```

Comment:
Could you explain to me how can I test it ? 
+ I didn't find any ".media_iframe_video" class in the DOM, and I think it's not possible that this class still exists here.
(Check `selector: '.media_iframe_video',` in snippets.animations.js

Reply:
https://youtu.be/9mV2kS7uiSo here are the steps to test

Reply:
can you show me where the 'media_iframe_video' class is (in the DOM) when you close the popup because I dont see it..
`<iframe id="o_bg_video_iframe_6" class="o_bg_video_iframe fade show" frameborder="0" src="//www.youtube.com/embed/J76TiX-PElY?autoplay=1&amp;mute=1&amp;rel=0&amp;loop=1&amp;playlist=J76TiX-PElY&amp;controls=0&amp;fs=0&amp;modestbranding=1" style="height: 100%; top: 0px; width: 134.216%; left: -17.1082%;"></iframe>`

Reply:
![Screenshot-from-2021-01-11-16-52-02](https://user-images.githubusercontent.com/25147715/104176623-90879c00-542d-11eb-91dc-0a500ab08bd4.png)

Reply:
Could you check in the following documentation if its possible to do something like
```
  video.pause();
  video.currentTime = 0;
```
It will be cleaner and easier to understand

https://developer.mozilla.org/fr/docs/Web/HTML/Element/video

Reply:
Here is a suggestion on bootstrap regarding handling youtube video https://getbootstrap.com/docs/4.0/components/modal/#embedding-youtube-videos I tried to follow the same.

Besides, `document.querySelectorAll('iframe')` does not fetch video element from the iframe, Could you suggest more on this?

Reply:
ok so dont change it ;)